agenta-backend/agenta_backend/services/selectors.py

```diff
@@ -27,4 +27,9 @@ async def get_user_objectid(user_id: str) -> Tuple[str, str]:
     """
 
     user = await users.find_one({"id": user_id})
-    return str(user["id"]), str(user["organization_id"])
+
+    if user:
+        user_id = str(user["id"])
+        organization_ids = [str(org.id) for org in user.organizations] if user.organizations else []
```

Comment:
How is this possible? 🤔 

You're currently using Pymongo to find a user in the db. What you will get is a dictionary, not an object.